### ISSUE TITLE
👥 Inform customers when changes are made to their `environments/*.json`

### ISSUE THREAD

**Issue by dms1981** [ministryofjustice/modernisation-platform#7402]
### User Story

As a Modernisation Platform Customer
I want to be notified when my `environments/*.json` file is part of a pull request
So that I am aware of any changes

### Value / Purpose

Our `environments/*.json` files provide information that our automation uses for a number of purposes; SSO access, GitHub reviewers, and OIDC repository permissions.

It's entirely possible that multiple teams are involved in the same environment; contractors delivering functionality, or a Ministry of Justice team supporting users. So that all stakeholders are aware when changes are made we want to find a way to inform the infrastructure support team when changes are made. This might be by simply ensuring that `./github/CODEOWNERS` references owning teams with the relevant files, or by setting up a GitHub action that notifies a listed slack channel in the `environments/*.json` file.

### Useful Contacts

@dms1981, @georgepstaylor

### Additional Information

This might actually be a spike, but the outcome should be clear enough. Find a way to tell a team when their environments json has been changed or is in a PR.

If we choose Slack as a notification method, and customers do not supply a Slack channel, then they should see this as an incentive to set up a Slack channel for notifications.

### Definition of Done

- [ ] Identify options to notify team (Slack? Codeowners?)
- [ ] Implement the most suitable option
- [ ] Test the implementation with something owned by the `modernisation-platform` team
- [ ] Update any supporting documentation (new account process?)

**Comment by sukeshreddyg** [comment]
For this ticket, I plan to implement a solution where, when there is a change to a particular JSON file, a notification will be sent to the infrastructure team's email stating that the corresponding JSON file was changed. This notification will be sent once the change is merged into the main branch.

**Comment by sukeshreddyg** [comment]
After discussing with the team, we plan to implement two solutions. One is to send a notification to the `modernisation-platform-update` channel, and the other is to send an email to the `infrastructure team's email`.
